metagraph/src/cli/config/config.cpp

```diff
@@ -344,6 +344,8 @@ Config::Config(int argc, char *argv[]) {
             tmp_dir = get_value(i++);
         } else if (!strcmp(argv[i], "--disk-cap-gb")) {
             disk_cap_bytes = atoi(get_value(i++)) * 1e9;
+        } else if (!strcmp(argv[i], "--anchors-file")) {
+            anchor_fname = get_value(i++);
```

Comment:
we usually don't write "file" in flags, so just `--anchors` will be more consistent with the rest.